code/modules/mob/living/living_defines.dm

```diff
@@ -139,3 +139,6 @@
 
 	/// This is the cooldown on suffering additional effects for when we exhaust all stamina
 	COOLDOWN_DECLARE(last_stamina_exhaustion)
+
+	var/lastgundelay //This is the time after which the next gun can be fired in world time.
```

Comment:
Convention is to use _ between words.
Also that is not how autodocing is, it is three / and one line before

Reply:
Sigh I copied documentation from other vars but sure. I'll fix it once I'm home.

Reply:
Fixed.